Source/SG/SG.cs

```diff
@@ -24,17 +24,18 @@ public struct InstrumentConfiguration
             public double CarrierFrequency_Hz;
             public double AverageInputPower_dBm;
             public double ExternalAttenuation_dBm;
-            public string BurstStartTriggerExportTerminal;
             public bool ShareLOSGToSA;
-            public void SetDefaults()
+        }
+        public static InstrumentConfiguration GetDefaultInstrumentConfiguration()
+        {
+            return new InstrumentConfiguration
             {
-                ReferenceClockSource = RfsgFrequencyReferenceSource.PxiClock.ToString();
-                CarrierFrequency_Hz = 1e9;
-                AverageInputPower_dBm = 0;
-                ExternalAttenuation_dBm = 0;
-                BurstStartTriggerExportTerminal = RfsgMarkerEventExportedOutputTerminal.PxiTriggerLine0.ToString();
-                ShareLOSGToSA = true;
-            }
+                ReferenceClockSource = RfsgFrequencyReferenceSource.PxiClock.ToString(),
+                CarrierFrequency_Hz = 1e9,
+                AverageInputPower_dBm = 0,
+                ExternalAttenuation_dBm = 0,
+                ShareLOSGToSA = true,
```

Comment:
why default true instead of false?

Reply:
The typical use case is PA/FEM DUTs where shared LO is the recommended configuration.